[3.1] Grid, Treelist, Gantt - customize filtering docs (#810)
telerik/blazor-docs: common-features/filter-operators.md

---
title: Filter Operators
page_title: Filter Operators
description: Supported Filter Operators in the Telerik UI for Blazor component suite.
slug: common-features-filter-operators
tags: telerik,blazor,filter,operator
published: True
position: 10
---

# Filter Operators

The `FilterOperator` enum contains all filter operators supported in the Telerik UI for Blazor suite.

> When customizing the filtering behavior of the components, it is important to use the correct filter operators depending on the specific data type. Otherwise, you will receive an exception - the assigned `DefaultFilterOperator` not applicable for the current data type.

Reviewer: Consider different text logic, which allows shorter sentences. For example something like this:

```suggestion
Each filter operator is compatible with a specific data type. Always use the correct filter operator during programmatic filter customizations. Otherwise, an exception will occur about the assigned `DefaultFilterOperator` not applicable for the current data type.
```
